net-p2p/tremc/tremc-0.9.2.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8} )
+PYTHON_REQ_USE="ncurses"
+inherit bash-completion-r1 python-single-r1
+
+DESCRIPTION="Ncurses interface for the Transmission BitTorrent client"
+HOMEPAGE="https://github.com/tremc/tremc"
+SRC_URI="https://github.com/tremc/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="geoip"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="${PYTHON_DEPS}
+	$(python_gen_cond_dep '
+		geoip? ( dev-python/geoip-python[${PYTHON_USEDEP}] )
+	')
+"
```

Comment:
Should transmission be added as a dep here?

Reply:
I don't think so, as it can be used to connect to a remote Transmission. If one is not running on localhost, it just exits with a complaint that the connection was refused.

Reply:
Good point, and thinking.